src/Bicep.LangServer/BicepCompilationManager.cs

```diff
@@ -465,21 +469,7 @@ private void SendTelemetryOnBicepFileOpen(SemanticModel semanticModel, DocumentU
             properties.Add("Resources", sematicModel.DeclaredResources.Length.ToString());
             properties.Add("Variables", declarationsInMainFile.Count(x => x is VariableDeclarationSyntax).ToString());
 
-            var localPath = bicepFile.FileUri.LocalPath;
```

Comment:
Tying language server implementation to the file system felt unnecessary so I opted to instead just return a character count here.

Reply:
You could get byte length of contents, but probably not important.